src/libAtomVM/linkedlist.h

```diff
@@ -127,13 +127,13 @@ static inline void linkedlist_prepend(struct ListHead **list, struct ListHead *n
  * @details Returns the length of a linked list
  * @param list a pointer to the linked list
  */
-static inline size_t linkedlist_length(struct ListHead *list)
+static inline size_t linkedlist_length(const struct ListHead *list)
```

Comment:
This was added from a comment in the previous PR.  Thought I'd add it here.

Reply:
I agree with this change, but I would have rather did this on a different PR.

Reply:
See https://github.com/bettio/AtomVM/pull/123